ci/requirements-ac.txt

```diff
@@ -1,30 +1,38 @@
 # use update-requirements.py to update this file
 
+certifi==2020.6.20        # via matplotlib
+click==7.1.2              # via nltk
 cycler==0.10.0            # via matplotlib
 decorator==4.4.2          # via networkx
 editdistance==0.5.3       # via -r tools/accuracy_checker/requirements.in
+fast-ctc-decode==0.2.5    # via -r tools/accuracy_checker/requirements.in
 imageio==2.9.0            # via scikit-image
-joblib==0.14.1            # via scikit-learn
-kiwisolver==1.1.0         # via matplotlib
-matplotlib==3.0.3         # via scikit-image
-networkx==2.4             # via scikit-image
-nibabel==3.0.2            # via -r tools/accuracy_checker/requirements.in
-numpy==1.17.5             # via -r tools/accuracy_checker/requirements-core.in, imageio, matplotlib, nibabel, pywavelets, scikit-learn, scipy
-pillow==7.2.0             # via -r tools/accuracy_checker/requirements.in, imageio, scikit-image
-py-cpuinfo==4.0.0         # via -r tools/accuracy_checker/requirements.in
+joblib==0.17.0            # via nltk, scikit-learn
+kiwisolver==1.2.0         # via matplotlib
+matplotlib==3.3.2         # via scikit-image
+networkx==2.5             # via scikit-image
+nibabel==3.1.1            # via -r tools/accuracy_checker/requirements.in
+nltk==3.5                 # via -r tools/accuracy_checker/requirements.in
+numpy==1.17.5             # via -r tools/accuracy_checker/requirements-core.in, imageio, matplotlib, nibabel, parasail, pywavelets, scikit-image, scikit-learn, scipy, tifffile
+packaging==20.4           # via nibabel
+parasail==1.2             # via -r tools/accuracy_checker/requirements.in
+pillow==8.0.0             # via -r tools/accuracy_checker/requirements-core.in, imageio, matplotlib, scikit-image
```

Comment:
lets wait a one day, when validation is finished. pillow 8.0.0 was released only today. I want make sure that it works as expected.

Reply:
> pillow 8.0.0 was released only today.

What a coincidence. :) Sure, I can wait.